.deployment/deploy.yml

```diff
@@ -0,0 +1,261 @@
+
+    - name: create application directory
+      become: true
+      file:
+        path: /opt/tobira/{{ id }}
+        state: directory
+        owner: root
+        group: root
+        mode: '0755'
+
+    - name: create socket directory
+      become: true
+      file:
+        path: /opt/tobira/{{ id }}/socket
+        state: directory
+        owner: tobira
+        group: tobira
+        setype: httpd_sys_rw_content_t
+        mode: '0755'
+
+    - name: deploy tobira and configuration
+      become: true
+      copy:
+        src: tobira
+        dest: /opt/tobira/{{ id }}/
+        owner: root
+        group: root
+        mode: '0755'
+      notify: restart tobira
+
+    - name: deploy configuration
+      become: true
+      template:
+        src: config.toml
+        dest: /opt/tobira/{{ id }}/
+        owner: root
+        group: root
+        mode: '0644'
+      notify: restart tobira
+
+    - name: deploy demo data
+      become: true
+      copy:
+        src: fixtures.sql
+        dest: /opt/tobira/{{ id }}/
+        owner: root
+        group: root
+        mode: '0644'
+
+    - name: run database migration
+      become: true
+      become_user: tobira
+      command:
+        cmd: /opt/tobira/{{ id }}/tobira db migrate
+        chdir: /opt/tobira/{{ id }}/
+
+    - name: insert demo data
+      become: true
+      become_user: tobira
+      command:
+        cmd: /opt/tobira/{{ id }}/tobira db script fixtures.sql
+        chdir: /opt/tobira/{{ id }}/
+
+    - name: install tobira service files
+      become: true
+      template:
+        src: tobira.service
+        dest: /etc/systemd/system/tobira-{{ id }}.service
+        mode: '0644'
+        owner: root
+        group: root
+      notify: restart tobira
+
+    - name: start and enable tobira
+      become: true
+      systemd:
+        daemon_reload: true
+        name: tobira-{{ id }}
+        state: started
+        enabled: true
+
+    # NGINX
+
+    - name: deploy nginx configuration
+      become: true
+      template:
+        src: '{{ item.src }}'
+        dest: /etc/nginx/{{ item.dest }}
+        mode: '0644'
+        owner: root
+        group: root
+      loop:
+        - src: nginx.conf
+          dest: nginx.conf
+        - src: nginx-host.conf
+          dest: conf.d/tobira-{{ id }}.conf
+      notify: reload nginx
+
+    - name: create ssl directory
+      become: true
+      file:
+        path: /etc/nginx/ssl
+        state: directory
+        owner: nginx
+        mode: '0755'
+
+    - name: initial configuration for certificates
+      become: true
+      copy:
+        src: ssl/{{ item.src }}
+        dest: /etc/nginx/ssl/{{ item.dest }}
+        owner: root
+        group: root
+        force: false
+        mode: '0640'
+      loop:
+        - src: certificate.pem
+          dest: '{{ inventory_hostname }}.crt'
+        - src: key.pem
+          dest: '{{ inventory_hostname }}.key'
+        - src: dhparam.pem
+          dest: dhparam.pem
```

Comment:
Long story short: You should deploy a proper one. It does not necessary has to be unique. You can generate it or pick a good one. And yes, this whole step is just to deploy something to work with in case nothing better is deployed.

Reply:
Long story short: You should deploy a proper one. It does not necessary has to be unique. You can generate it or pick a good one. And yes, this whole step is just to deploy something to work with in case nothing better is deployed.